packages/satusehat/src/Adaptor.js

```diff
@@ -142,7 +154,7 @@ export function put(path, data, params = {}, callback = s => s) {
 
 /**
  * Make a PATCH request to Satusehat
- * @example
+ * @example <caption>Make a PATCH request on an Organization</caption>
```

Comment:
What does a PATCH request do? The example would be clearer if it read like "Make a PATCH request to update an organisation", or whatever the correct verb is